tests/src/actionContainers/NodeJsActionContainerTests.scala

```diff
@@ -444,7 +474,7 @@ class NodeJsActionContainerTests extends BasicActionRunnerTests with WskActorSys
         checkStreams(out, err, {
             case (o, e) =>
                 (o + e).toLowerCase should include("error")
-                (o + e).toLowerCase should include("package.json must be located at the root of a zipped action")
+                (o + e).toLowerCase should include("zipped actions must contain either package.json or index.js at the root.")
```

Comment:
It's a good question. The simplest answer, I suppose, is to insulate the test (a little) from simple changes in capitalization.

 I was just going to let the sleeping dog lie though...